### ISSUE TITLE
Package is not immutable

### ISSUE THREAD

**Issue by AlexaCodex** [chriseldredge/Klondike#202]
Not sure why but i am able to **push same version with different code(dll)** to our Klondike server, which makes it mutable. How to stop this behaviour?

**Comment by AlexaCodex** [comment]
Thanks, will give it a go tomorrow. 
Chris - Is the project in maintenance mode now, as i don't see any new check ins?

**Comment by AlexaCodex** [comment]
I can confirm the change suggested above, worked as expected.

**Comment by chriseldredge** [comment]
Yes, the project is in maintenance mode. The .net package management ecosystem has moved on from what Klondike supports and it would be a major undertaking to try to bring it up to date. On a personal note I also have no current need, or expected future need for Klondike.
